inst/rstudio/templates/project/RcppParallel.package.skeleton.dcf

```diff
@@ -0,0 +1,10 @@
+Binding: RcppParallel.package.skeleton
+Title: R Package using RcppParallel
+Subtitle: Create a new R Package using RcppParallel
+Caption: Create R Package using RcppParallel
+OpenFiles: Read-and-delete-me, src/vector-sum.cpp
```

Comment:
I prefer to have Read-and-delete-me type content within comments of source files, but if you think the extra file is warranted here I'm certainly okay with it.